test/integration/targets/nxos_igmp_interface/tests/common/sanity.yaml

```diff
@@ -56,7 +70,78 @@
       that:
         - "result.changed == false"
 
-  - name: Configure igmp interface with default value
+  - name: Configure igmp interface with some default values
+    nxos_igmp_interface: &sdef
+      interface: "{{ intname }}"
+      version: default
+      startup_query_interval: default
+      startup_query_count: default
+      robustness: default
+      querier_timeout: default
+      query_mrt: default
+      query_interval: default
+      last_member_qrt: default
+      last_member_query_count: default
+      group_timeout: default
+      oif_ps:
+        - {'prefix': '238.2.2.6'}
+        - {'prefix': '238.2.2.5'}
+        - {'source': '1.1.1.1', 'prefix': '238.2.2.5'}
```

Comment:
Please no 1.1.1.1 use the documention ip ranges

Reply:
Changed 1.1.1.1 to 192.168.0.1 (not sure if that is what you meant). 
However, the device takes 1.1.1.1 so the ip itself is correct.
========
see below:
==========
n9k-108(config-if)# sh run igmp

!Command: show running-config igmp
!Time: Fri Apr 20 14:55:54 2018

version 7.0(3)I7(2)

interface Ethernet1/1
  ip igmp static-oif 239.2.2.2 source 1.1.1.1